var/spack/repos/builtin/packages/singularityce/package.py

```diff
@@ -190,7 +190,7 @@ class Singularityce(SingularityBase):
     See package definition or `spack-build-out.txt` build log for details,
     e.g.
 
-    tail -15 $(spack location -i singularity)/.spack/spack-build-out.txt
```

Comment:
Are you sure the executable is now named `singularityce`? Looking at the documentation it seems it is still called `singularity`, see e.g. [here](https://docs.sylabs.io/guides/main/user-guide/quick_start.html#interacting-with-images).